<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Witaj </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="test">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>241</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>menu klijnij w wybrana opcje</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>130</y>
         <width>472</width>
         <height>69</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Przelew</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Historia</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Konto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Wyloguj</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>541</width>
         <height>311</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Numer konta</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="acc_id"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Tytul</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="title"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>kwota</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="amount"/>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="commandLinkButton">
          <property name="text">
           <string>Wyslij</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>wroc</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>270</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Wroc</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>konto</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>220</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>370</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Wroc</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>451</width>
         <height>321</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>